tx/textual/valuerenderer/valuerenderer.go

```diff
@@ -84,8 +84,13 @@ func (r Textual) GetValueRenderer(fd protoreflect.FieldDescriptor) (ValueRendere
 
 		vr, found := r.messages[fullName]
 		if found {
-			return vr, nil
+			if fd.IsList() {
+				return NewCoinsValueRenderer(r.coinMetadataQuerier, fd), nil
+			} else {
+				return vr, nil
+			}
 		}
```

Comment:
This is incorrect, but I need to think a bit about how to find a solution for this.